### PR TITLE
Add firewall statistic collector

### LINKED ISSUE #55: Implement firewall statistics
The statistics for each firewall rules which include packet drops can be retrieved from `client.ServicesApi.GetFirewallStats`.

### PR DESCRIPTION
**Pull request by girikuncoro**
This collects firewall rule statistics by iterating
through firewall sections and rules. Metrics include
packets and bytes processed by each rule.

Signed-off-by: Giri Kuncoro <girikuncoro@gmail.com>

### CODE REVIEW

File: client/nsxt_client.go [hunk 1]
```diff
@@ -293,3 +293,46 @@ func (c *nsxtClient) GetLoadBalancerStatus(loadBalancerID string) (loadbalancer.
 	loadBalancerStatus, _, err := c.apiClient.ServicesApi.ReadLoadBalancerServiceStatus(c.apiClient.Context, loadBalancerID, nil)
 	return loadBalancerStatus, err
 }
+
+func (c *nsxtClient) ListAllSections() ([]manager.FirewallSection, error) {
```
Comment (walbertus) [inline client/nsxt_client.go]: Is it better to name it ListAllFirewallSections?

File: client/nsxt_client.go [hunk 2]
```diff
@@ -293,3 +293,46 @@ func (c *nsxtClient) GetLoadBalancerStatus(loadBalancerID string) (loadbalancer.
...
+		firewallSections = append(firewallSections, firewallSectionsResult.Results...)
+		cursor = firewallSectionsResult.Cursor
+		if len(cursor) == 0 {
+			break
+		}
+	}
+	return firewallSections, nil
+}
+
+func (c *nsxtClient) GetAllRules(sectionID string) ([]manager.FirewallRule, error) {
```
Comment (walbertus) [inline client/nsxt_client.go]: This method name to GetAllFirewallRules as we have multiple rules

File: client/nsxt_client.go (hunk 1 above)
Comment (girikuncoro) [reply client/nsxt_client.go]: fixed in 8aef830

File: client/nsxt_client.go (hunk 2 above)
Comment (girikuncoro) [reply client/nsxt_client.go]: fixed in 8aef830

**Review by walbertus — APPROVED** [review]
LGTM

**Pull request merged**